packages/empty-state/src/styledComponents/Container.js

```diff
@@ -0,0 +1,30 @@
+// @flow
+import styled from "styled-components";
+
+const width = props => props.width;
+
+const Container = styled.div`
+  width: ${width}px;
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  -webkit-font-smoothing: antialiased;
+
+  > .thumbnail {
+    margin-bottom: 24px;
```

Comment:
Correct margin-bottom is `16px`
![image](https://user-images.githubusercontent.com/17936244/33076617-55ac23fa-ceb4-11e7-8a87-ac772074fe2b.png)

Reply:
Ok... nevermind